### PR TITLE
[KEW-1715] Intalation via npm and yarm

### PR DESCRIPTION
**Pull request by islasjuanp**
Provided yarn and npm installation ability and instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ Javascript API for integration with the WAX Cloud Wallet.
...
+### NPM
+
+```bash
+npm install waxjs
+```
+
+### YARN
+
+```bash
+yarn add eosjs
```
Comment (porkchop) [inline README.md]: `yarn add waxjs`?

File: package.json [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
     "docs": "typedoc --excludePrivate --exclude src/waxEventSource.ts --out docs src",
-    "serve": "serve -s dist-web"
+    "serve": "serve -s dist-web",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm test && npm run lint",
+    "preversion": "npm run lint",
+    "version": "npm run prettier && git add -A src",
+    "postversion": "git push && git push --tags"
   },
   "author": "keyboard-warriors",
```
Comment (porkchop) [inline package.json]: we should change the author to `WAX`

File: README.md (hunk 1 above)
Comment (islasjuanp) [reply README.md]: Fixed :)

File: package.json (hunk 2 above)
Comment (islasjuanp) [reply package.json]: Fixed :)

**Pull request merged**